brainwallet/Bento Effects /ThemeGradientHelpers.swift

```diff
@@ -0,0 +1,167 @@
+                                                          Color(red: 0.5254901960784314,
+                                                                green: 0.4117647058823529,
+                                                                blue: 0.7294117647058823).opacity(0.07)],
+                                                                   startPoint: .topLeading, endPoint: .bottomTrailing) :
+                                LinearGradient(
+                                    colors: [.white, Color(red: 0.91764, green: 0.91764, blue: 0.91764)],
+                                    startPoint: .top,
+                                    endPoint: .bottom
+                                )
+            )
+
+        /// Dark Mode : #000000 at 40% Opacity
+        /// Light  Mode : #5754FF1F at 12% Opacity
+    }
+}
+
+struct BentoBackgroundView: View {
+
+    @Binding
+    var userPrefersDarkTheme: Bool
+
+    var body: some View {
+        ZStack {
+            if userPrefersDarkTheme {
+                LinearGradient(colors: [Color(red: 0.5254901960784314,
+                                         green: 0.4117647058823529,
+                                         blue: 0.7294117647058823).opacity(0.25),
+                                        Color(red: 0.5254901960784314,
+                                              green: 0.4117647058823529,
+                                              blue: 0.7294117647058823).opacity(0.07)],
+                               startPoint: .topLeading, endPoint: .bottomTrailing)
+                                        .edgesIgnoringSafeArea(.all)
+                RoundedRectangle(cornerRadius: bentoCornerRadius)
+                    .stroke( LinearGradient(
+                        colors: [Color(red: 0.7843137254901961,
+                                       green: 0.7019607843137254,
+                                       blue: 0.9333333333333333)// #C8B3EE
+                                 ,Color(red: 0.5254901960784314,
+                                        green: 0.4117647058823529,
+                                        blue: 0.7294117647058823)], // #8669BA80 50%
+                        startPoint: .leading,
+                        endPoint: .trailing
+                    ),lineWidth: 1.5)
+            } else {
+                LinearGradient(
+                    colors: [.white, Color(red: 0.91764, green: 0.91764, blue: 0.91764)],
+                    startPoint: .top,
+                    endPoint: .bottom
+                ).edgesIgnoringSafeArea(.all)
+                RoundedRectangle(cornerRadius: bentoCornerRadius)
+                    .stroke(Color(red: 0.9098039215686274,
+                                  green: 0.9176470588235294,
+                                  blue: 0.9254901960784314),
+                            lineWidth: 1.5)
+            }
+        }
+    }
+}
+
+struct BalanceGameBackgroundView: View {
+
+    @Binding
```

Comment:
Since the Balance and Game Bentos are different, using a different set of colors from the Figma @JosephSanjaya >
We can polish in 10 days